apps/nextjs-website/src/components/Form.tsx

```diff
@@ -1,13 +1,11 @@
 'use client';
-import MarkdownRenderer from './MarkdownRenderer';
 import { Form as FormRC } from '@react-components/components';
 import { FormProps } from '@react-components/types';
 import { FormSection } from '@/lib/fetch/types/PageSection';
 import { FormData } from '@react-components/types/Form/Form.types';
 
-const makeFormProps = ({ privacyText, ...rest }: FormSection): FormProps => ({
+const makeFormProps = ({ ...rest }: FormSection): FormProps => ({
   ...rest,
-  privacyText: MarkdownRenderer({ markdown: privacyText, variant: 'body2' }),
```

Comment:
Will neither `privacyLinkTextRecaptchaPolicy` nor `privacyLinkTextRecaptchaTerms` need to be rendered from Markdown?

Aka, will neither of them ever need to have bold or italic text, paragraphs or links inside the text's copy?